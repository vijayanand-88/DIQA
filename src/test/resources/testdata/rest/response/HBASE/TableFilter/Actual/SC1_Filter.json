{
  "count" : 3,
  "results" : [ {
    "id" : "Default.Table:::8102",
    "type" : "Table",
    "name" : "emp",
    "asg.parentTypeNamePath" : [ "Cluster/Sandbox/Service/HBASE/Database/pyspark" ],
    "asg.tagNames" : [ {
      "id" : "Default.tag:::20446",
      "name" : "SC1_onlyNamespace",
      "rootTagName" : "SC1_onlyNamespace",
      "parentTagName" : "",
      "definition" : "",
      "status" : "APPROVED",
      "type" : "Structure Information Tag",
      "userAssignable" : true,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-10-20T19:21:49.550+0000",
      "asg_createdat" : "2020-10-20T19:21:49.550+0000",
      "isRoot" : true,
      "catalog" : "Default",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 6658,
      "asg_modifiedby" : "Service",
      "target" : 8102
    }, {
      "id" : "Default.tag:::20452",
      "name" : "HBase",
      "rootTagName" : "Technology",
      "parentTagName" : "",
      "definition" : "HBase Distributed Database",
      "status" : "APPROVED",
      "type" : "Technology Tag",
      "image" : {
        "icon" : "icon-technology",
        "color" : "#35906c"
      },
      "userAssignable" : false,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-10-20T19:21:49.550+0000",
      "asg_createdat" : "2020-10-20T19:21:49.550+0000",
      "isRoot" : false,
      "catalog" : "Default",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 393,
      "asg_modifiedby" : "Service",
      "target" : 8102
    } ],
    "catalog" : {
      "name" : "Default",
      "label" : "Default"
    }
  }, {
    "id" : "Default.Table:::8101",
    "type" : "Table",
    "name" : "employee",
    "asg.parentTypeNamePath" : [ "Cluster/Sandbox/Service/HBASE/Database/pyspark" ],
    "asg.tagNames" : [ {
      "id" : "Default.tag:::20447",
      "name" : "SC1_onlyNamespace",
      "rootTagName" : "SC1_onlyNamespace",
      "parentTagName" : "",
      "definition" : "",
      "status" : "APPROVED",
      "type" : "Structure Information Tag",
      "userAssignable" : true,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-10-20T19:21:49.550+0000",
      "asg_createdat" : "2020-10-20T19:21:49.550+0000",
      "isRoot" : true,
      "catalog" : "Default",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 6658,
      "asg_modifiedby" : "Service",
      "target" : 8101
    }, {
      "id" : "Default.tag:::20453",
      "name" : "HBase",
      "rootTagName" : "Technology",
      "parentTagName" : "",
      "definition" : "HBase Distributed Database",
      "status" : "APPROVED",
      "type" : "Technology Tag",
      "image" : {
        "icon" : "icon-technology",
        "color" : "#35906c"
      },
      "userAssignable" : false,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-10-20T19:21:49.550+0000",
      "asg_createdat" : "2020-10-20T19:21:49.550+0000",
      "isRoot" : false,
      "catalog" : "Default",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 393,
      "asg_modifiedby" : "Service",
      "target" : 8101
    } ],
    "catalog" : {
      "name" : "Default",
      "label" : "Default"
    }
  }, {
    "id" : "Default.Table:::8103",
    "type" : "Table",
    "name" : "hremployee",
    "asg.parentTypeNamePath" : [ "Cluster/Sandbox/Service/HBASE/Database/pyspark" ],
    "asg.tagNames" : [ {
      "id" : "Default.tag:::20448",
      "name" : "SC1_onlyNamespace",
      "rootTagName" : "SC1_onlyNamespace",
      "parentTagName" : "",
      "definition" : "",
      "status" : "APPROVED",
      "type" : "Structure Information Tag",
      "userAssignable" : true,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-10-20T19:21:49.550+0000",
      "asg_createdat" : "2020-10-20T19:21:49.550+0000",
      "isRoot" : true,
      "catalog" : "Default",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 6658,
      "asg_modifiedby" : "Service",
      "target" : 8103
    }, {
      "id" : "Default.tag:::20454",
      "name" : "HBase",
      "rootTagName" : "Technology",
      "parentTagName" : "",
      "definition" : "HBase Distributed Database",
      "status" : "APPROVED",
      "type" : "Technology Tag",
      "image" : {
        "icon" : "icon-technology",
        "color" : "#35906c"
      },
      "userAssignable" : false,
      "subTags" : [ ],
      "asg_modifiedat" : "2020-10-20T19:21:49.550+0000",
      "asg_createdat" : "2020-10-20T19:21:49.550+0000",
      "isRoot" : false,
      "catalog" : "Default",
      "asg_createdby" : "Service",
      "asg_isnsa" : false,
      "source" : 393,
      "asg_modifiedby" : "Service",
      "target" : 8103
    } ],
    "catalog" : {
      "name" : "Default",
      "label" : "Default"
    }
  } ],
  "configs" : {
    "defaultListView" : [ {
      "value" : "asg.parentTypeNamePath",
      "label" : "Hierarchy"
    }, {
      "value" : "asg.tagNames",
      "label" : "Tags"
    } ]
  },
  "configsMap" : {
    "*" : {
      "*" : "defaultListView"
    }
  },
  "facets" : {
    "asg.tagPathsHierarchy_ss" : {
      "label" : "Tags",
      "type" : "TREE",
      "values" : [ {
        "count" : 3,
        "value" : "SC1_onlyNamespace",
        "label" : "SC1_onlyNamespace",
        "checked" : true,
        "childValues" : [ ]
      }, {
        "count" : 3,
        "value" : "Technology",
        "label" : "Technology",
        "checked" : false,
        "childValues" : [ {
          "count" : 3,
          "value" : "Technology/Big Data",
          "label" : "Big Data",
          "checked" : false,
          "childValues" : [ {
            "count" : 3,
            "value" : "Technology/Big Data/Operational",
            "label" : "Operational",
            "checked" : false,
            "childValues" : [ {
              "count" : 3,
              "value" : "Technology/Big Data/Operational/HBase",
              "label" : "HBase",
              "checked" : false,
              "childValues" : [ ]
            } ]
          } ]
        } ]
      } ]
    },
    "type_s" : {
      "label" : "Metadata Type",
      "type" : "FLAT_LIST",
      "values" : [ {
        "count" : 3,
        "value" : "Table",
        "checked" : true
      }, {
        "count" : 1,
        "value" : "Database",
        "checked" : false
      } ]
    },
    "asg.parentTypeNamePathHierarchy_ss" : {
      "label" : "Hierarchy",
      "type" : "TREE",
      "values" : [ {
        "count" : 3,
        "value" : "Cluster/Sandbox",
        "label" : "Sandbox [Cluster]",
        "checked" : false,
        "childValues" : [ {
          "count" : 3,
          "value" : "Cluster/Sandbox/Service/HBASE",
          "label" : "HBASE [Service]",
          "checked" : false,
          "childValues" : [ {
            "count" : 3,
            "value" : "Cluster/Sandbox/Service/HBASE/Database/pyspark",
            "label" : "pyspark [Database]",
            "checked" : true,
            "childValues" : [ ]
          } ]
        } ]
      } ]
    },
    "asg_modifiedat_dt" : {
      "label" : "Last Modified",
      "type" : "FLAT_LIST",
      "values" : [ {
        "count" : 0,
        "value" : "(*,NOW-1YEAR]",
        "label" : "1 year and older",
        "checked" : false
      }, {
        "count" : 0,
        "value" : "(NOW-1YEAR,NOW-6MONTH]",
        "label" : "6 months up to 1 year",
        "checked" : false
      }, {
        "count" : 0,
        "value" : "(NOW-6MONTH,NOW-1MONTH]",
        "label" : "1 month up to 6 months",
        "checked" : false
      }, {
        "count" : 0,
        "value" : "(NOW-1MONTH,NOW-7DAY]",
        "label" : "1 week up to 1 month",
        "checked" : false
      }, {
        "count" : 0,
        "value" : "(NOW-7DAY,NOW-1DAY]",
        "label" : "1 day up to 1 week",
        "checked" : false
      }, {
        "count" : 3,
        "value" : "(NOW-1DAY,NOW]",
        "label" : "past 24 hours",
        "checked" : false
      } ]
    },
    "asg_modifiedby_s" : {
      "label" : "Modified by",
      "type" : "FLAT_LIST",
      "values" : [ {
        "count" : 3,
        "value" : "Service",
        "checked" : false
      } ]
    }
  }
}