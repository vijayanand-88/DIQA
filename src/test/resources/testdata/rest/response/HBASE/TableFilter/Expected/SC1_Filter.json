{
  "SC1": {
    "Tables": [
      "emp",
      "hremployee",
      "employee"
    ],
    "Cluster": [
      "Sandbox"
    ]
  },
  "SC2": {
    "Tables": [
      "employee",
      "employee",
      "employee",
      "employee",
      "employee"
    ],
    "hierarchy": [
      "Automation_QA",
      "Automation_QA1",
      "Automation_QA2",
      "default",
      "pyspark"
    ]
  },
  "SC3": {
    "Tables": [
      "employee",
      "hremployee"
    ],
    "hierarchy": [
      "pyspark"
    ]
  },
  "SC4": {
    "Tables": [
      "employee_test",
      "employee",
      "employee_test",
      "employee",
      "employee_test",
      "employee",
      "employee1"
    ],
    "hierarchy": [
      "Automation_QA2",
      "Automation_QA",
      "Automation_QA1"
    ]
  },
  "SC5": {
    "Tables": [
      "employee",
      "employee",
      "employee1"
    ],
    "hierarchy": [
      "default",
      "Automation_QA1"
    ]
  },
  "SC6": {
    "Tables": {
      "employee": {
        "column": [
          "professional data:designation",
          "professional data:salary",
          "personal data:city",
          "personal data:name"
        ]
      }
    },
    "hierarchy": [
      "default"
    ]
  },
  "SC7": {
    "Tables": {
      "employee": {
        "column": [
          "professional data:designation",
          "professional data:salary",
          "personal data:city",
          "personal data:name",
          "personal data:address",
          "professional data:DOJ",
          "professional data:hike"
        ]
      }
    },
    "hierarchy": [
      "default"
    ]
  }
}